integration_tests/src/main/python/hive_write_test.py

```diff
@@ -66,8 +66,8 @@ def do_write(spark, table_name):
         store_name = storage
         if storage.endswith("orc"):
             store_name = "ORC"
-        return "CREATE TABLE {} STORED AS {} AS SELECT * FROM {}".format(
-            table_name, store_name, data_table)
+        return ["CREATE TABLE {} STORED AS {} AS SELECT * FROM {}".format(
+            table_name, store_name, data_table)]
```

Comment:
Indicative that the API changed and should not have.  This change should be reverted.

Reply:
Agreed. There should be no change in this file now.